step/src/main/webapp/infoForm.jsp

```diff
@@ -0,0 +1,136 @@
+<%--
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+--%>
+<%@ page import="com.google.appengine.api.users.UserService" %>
+<%@ page import="com.google.appengine.api.users.UserServiceFactory" %>
+<% UserService userService = UserServiceFactory.getUserService();
+   String urlToRedirectToAfterUserLogsOut = "/";
+   String logoutURL = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut); %> 
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="style_pref.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <script src="script.js"></script>
+    <title>My Information Form </title>
+  </head>
+  <body>
+    <nav>
+        <a href="<%= logoutURL %>"> Log Out </a>
+        <h1 style="float: left; font-size: 25px"> Friend Matching Plus </h1>
+    </nav>
+    
+    
+    <% if (!userService.isUserLoggedIn()) {
+        response.sendRedirect("index.jsp");   
+    } %>
+    <section class="infoform">
+    <form id="regForm" action="Info" method="POST"> 
+        <div class="tab"> Name: 
+            <p><input placeholder="First Name" oninput="this.className = ''"  name="fname"></p>
+            <p><input placeholder="Last Name" oninput="this.className = ''"  name="lname"></p>
+        </div>
+        <div class="tab"> Contact: 
+            <p><input placeholder="Phone" oninput="this.className = ''" name="phone"></p>
+        </div>
+        <div class="tab"> Birthday
+            <p>Day<input placeholder="dd" oninput="this.className = ''" name="dbirth"></p>
+            <p>Month<input placeholder="mm" oninput="this.className = ''" name="mbirth"></p>
+            <p>Year<input placeholder="yy" oninput="this.className = ''" name="ybirth"></p>
+        </div>
+        <div style="overflow:auto;">
+            <div style="float:right;">
+                <button type="button" id="prevBtn" onclick="nextPrev(-1)"> Previous </button>
```

Comment:
Giving in 1 and -1 for the next and prev function isn't super readable/intuitive when scanning over the code. If there's a lot of repeated code between next and prev, maybe move the repeated bits to a helper and have separate next() and prev() functions which then call on the helper?

Reply:
Technically, the button can only move 1 step forward or backward at once. I can separate that into 2 different functions but I think it would be not really necessary because it functions in the same way